src/events/cd-event-add-child-ticket.vue

```diff
@@ -117,7 +124,7 @@
           dojoId: this.event.dojoId,
           ticketId: ticket.id,
           userId: this.userId,
-        }));
+        }, this.specialRequirement === 'no' ? '' : { specialRequirement: this.specialRequirement })));
```

Comment:
If you return a falsy value when not set, you wouldn't have to compare to a string ;)